src/Splunk.OpenTelemetry.AutoInstrumentation/Logging/Logger.cs

```diff
@@ -28,13 +28,13 @@ static Logger()
     {
         try
         {
-            var otelLoggingtype = Type.GetType("OpenTelemetry.AutoInstrumentation.Logging.OtelLogging, OpenTelemetry.AutoInstrumentation");
+            var otelLoggingtype = Type.GetType("OpenTelemetry.AutoInstrumentation.Logging.OtelLogging, OpenTelemetry.AutoInstrumentation")!;
```

Comment:
I am not sure if null forgiving operator should be used here.
Maybe better option will be to null check?

Reply:
If this library is missing, there will be no plugin instance anyway.

Reply:
... just need to be sure that the logging namespace exists. Preferably by some tests.